packages/core/src/App/Containers/SettingsModal/settings-language.jsx

```diff
@@ -21,7 +21,7 @@ const NonClickableLink = ({ children, lang }) => (
 const LanguageLink = ({ lang }) => (
     <React.Fragment>
         <Icon
-            icon='IcFlagUk' // TODO: enable other language
+            icon={`IcFlag${lang.indexOf('_') > -1 ? lang.split('_')[1] : lang}`} // some lang codes have 2 parts like ZH_CN, so we use the second part
```

Comment:
why don't we keep the full name to avoid having an exception? we can just replace `_` in it

Reply:
didn't think of that, thanks.